corehq/apps/accounting/usage.py

```diff
@@ -21,14 +21,19 @@ def __init__(self, feature_rate, domain_name,
         self.end_date = end_date or datetime.date(
             today.year, today.month, last_day)
 
+    @property
+    def usage_fns(self):
+        return {
+            FeatureType.USER: self._get_user_usage,
+            FeatureType.SMS: self._get_sms_usage,
+            FeatureType.WEB_USER: self._get_web_user_usage,
+            FeatureType.FORM_SUBMITTING_MOBILE_WORKER: self._get_form_submitting_mobile_worker_user_usage,
+        }
```

Comment:
It doesn't look like there is logic here, so it appears `usage_fns` could be a dictionary that is initialized in the constructor. What is the advantage of using a property here?

Reply:
Do you mean like this?
```python
    def __init__(self, feature_rate, domain_name,
                 start_date=None, end_date=None):
        ...
        self.usage_fns = {
            ...
        }
```
I felt a property was more readable, but only by a small amount.

Reply:
Yeah. I guess I have a bias against properties